File: PadelClub.Services/Database/ClubReview.cs
namespace PadelClub.Services.Database
{
    public class ClubReview
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public int Rating { get; set; }
        public string Comment { get; set; } = string.Empty;
        public bool IsPublished { get; set; } = true;
        public DateTime CreatedAt { get; set; } = DateTime.UtcNow;
        public DateTime? UpdatedAt { get; set; }

        public virtual User User { get; set; } = null!;
    }
}

File: UI/padelclub_desktop/lib/features/reviews/domain/entities/club_review.dart
class ClubReview {
  const ClubReview({
    required this.id,
    required this.userId,
    required this.memberName,
    required this.rating,
    required this.comment,
    required this.createdAt,
  });

  final int id;
  final int userId;
  final String memberName;
  final int rating;
  final String comment;
  final DateTime createdAt;
}
